webextension/scripts/popup.js

```diff
@@ -74,7 +74,18 @@ function loginSuccess() {
       if (isNotExcludedUrl(url)) {
         $('#savebtn').click(save_now)
         $('#contextTip').click(openContextMenu)
-        chrome.storage.local.get(['private_mode'], (event) => {
+        cached_url_data.forEach((value, key) => {
+          if(value === url) {
+            if(key === 'last_save') {
+              if($('#last_save').text !== 'URL not supported') {
+                $('#last_save').text(key);
+                break;
```

Comment:
While executing your code, I got this error in the browser console: `Uncaught SyntaxError: Illegal break statement`.  [More on break statement.](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/break)
This error is disorienting the DOM of the extension. You can see the following screenshot:

![mitchell-frost](https://user-images.githubusercontent.com/42881874/90489096-75548100-e15a-11ea-9d3a-2ada85a31acc.PNG)

**How to check for such errors?** 
For Chrome: You can right-click the extension icon in the browser toolbar and click on the "Inspect popup" option. This will open a developer tools window.

Reply:
Also, the variable `cached_url_data` is defined in `background.js` but is also being used in `popup.js`, where it can't be accessed.

Reply:
Thank You. I'll change the code accordingly, @tikhsuP